rmw_opendds_cpp/src/rmw_subscription.cpp

```diff
@@ -178,12 +164,12 @@ rmw_create_subscription(
   }
 
   // find or create DDS::Topic
-  DDS::TopicDescription_var topic_description = participant->lookup_topicdescription(topic_str.c_str());
-  DDS::Topic_var topic = topic_description ?
-    participant->find_topic(topic_str.c_str(), DDS::Duration_t{0, 0}) :
-    participant->create_topic(topic_str.c_str(), type_name.c_str(), TOPIC_QOS_DEFAULT, NULL, OpenDDS::DCPS::NO_STATUS_MASK);
+  DDS::TopicDescription_var topic_des = node_info->participant->lookup_topicdescription(topic_str.c_str());
```

Comment:
One good rule of thumb I remember for variable names is to use something that's pronounceable. With modern IDEs I don't see value in shorter names since we have auto completion and convenient ways to reference the names throughout the code. My suggestion is to switch this back to the original name of `topic_description`. It's a lot more meaningful.